site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -3431,6 +3431,7 @@ private function getMarkStats(Mark $mark, User $grader) {
     private function getStats(Gradeable $gradeable, User $grader, bool $full_stats, &$total_graded, &$total_total) {
         $num_components = $this->core->getQueries()->getTotalComponentCount($gradeable->getId());
         $sections = [];
+    
```

Comment:
Whitespace additions should be reverted, as they are outside the scope of this PR, and some of them will cause phplint failures.